Libraries/src/Amazon.Lambda.APIGatewayEvents/APIGatewayProxyRequest.cs

```diff
@@ -234,10 +234,11 @@ public class ProxyRequestContext
             /// Gets and sets the operation name.
             /// </summary>
             public string OperationName { get; set; }
-            
+
             /// <summary>
             /// Gets and sets the error.
             /// </summary>
+            [Obsolete("This property is obsolete since it is not processed by the Velocity Template Language engine and is not available as a field to integration request.")]
```

Comment:
is there a non obsolete field to recommend instead and if so can you mention that

Reply:
I suspect most users wouldn't know the context of "Velocity Template", I had to look it up myself. I suggest keeping the error message a bit more generic that it just isn't used by the service.

```
This property is obsolete since it is not used by AWS API Gateway and will never have value set by the service.
```